boards/xtensa/lilygo_t3/doc/index.rst

```diff
@@ -0,0 +1,216 @@
+.. _liligo_t3:
+
+LILYGO T3
```

Comment:
Are you sure that's the "official" and full name of this board? I can't seem to get many hits on Google when searching for "Lilygo T3". I wonder if this should be "Lilygo TTGO T3"? And it also looks like there are multiple revisions, so you might need to address that too (looks like you ported v1.6.1?)

Reply:
Hi kartben,
as I replied on discord:
The company's name is lilygo (www.lilygo.cc) on their github page, all products seem to have a TTGO prefix, on their homepage the "Lilygo TTGO T3" is sold as the "LoRa32 V2_1.6", pictures show the "T3_V1.6.1 ". I have four of them, ordered in three shipments, and they all are V1.6.1.
 I'm confused.

Reply:
Ya I am super confused too. I guess we could switch to "Lilygo TTGO T3"? But that's still not a name that seems to give an obvious "hit" on Google... Should this be deemed a better (less worse?) name (and I am really not saying it is at this point), then all lilygo_t3 files/folders should be renamed, as well as mentions in the index.rst etc.

Reply:
Hi Ben, 
I renamed lilygo_t3 to lilygo_ttgo_t3.
Please review the changes